### PR TITLE
Prepare next monthly update

### PR DESCRIPTION
**Pull request by dholbach**
Fix script for December.

Signed-off-by: Daniel Holbach <daniel@weave.works>

### CODE REVIEW

**Comment by dholbach** [comment]
We need to figure out if we can get an update out for December (I won't have time).

Or we combine the Dec+Jan update.

File: content/en/blog/2023-01-01-december-update/index.md [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
```
Comment (dholbach) [inline content/en/blog/2023-01-01-december-update/index.md:85]: @chanwit Anything we should bring up here for Dec+Jan?

File: content/en/blog/2023-01-01-december-update/index.md [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+#### Terraform-controller
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.15.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.14.1>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.13.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.12.0>
+
```
Comment (dholbach) [inline content/en/blog/2023-01-01-december-update/index.md:94]: @jamwils @makkes  Anything we should bring up here for Dec+Jan?

File: content/en/blog/2023-01-01-december-update/index.md [hunk 3]
```diff
@@ -0,0 +1,330 @@
...
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```
Comment (dholbach) [inline content/en/blog/2023-01-01-december-update/index.md:98]: @juozasg  Anything we should bring up here for Dec+Jan?

File: content/en/blog/2023-01-01-december-update/index.md [hunk 4]
```diff
@@ -0,0 +1,330 @@
...
+#### New additions to the Flux Ecosystem
+
+## Recent & Upcoming Events
+
```
Comment (dholbach) [inline content/en/blog/2023-01-01-december-update/index.md:102]: @staceypotter @vanessaabankwah Anything we should bring up here for Dec+Jan+Feb?

File: content/en/blog/2023-01-01-december-update/index.md (hunk 2 above)
Comment (JamWils) [reply content/en/blog/2023-01-01-december-update/index.md:94]: @dholbach can you reach out to the other PMs as well?  I am out for the next two weeks.

Comment (dholbach) [reply content/en/blog/2023-01-01-december-update/index.md:94]: Will do.

Comment (dholbach) [reply content/en/blog/2023-01-01-december-update/index.md:94]: @josefaworks Do you know who could help writing a bit of a Dec+Jan summary about what happened in Weave GitOps Core and Enterprise?

Here are past examples: https://fluxcd.io/blog/2022/12/november-2022-update/#flux-ecosystem and https://fluxcd.io/blog/2022/11/october-2022-update/#flux-ecosystem and https://fluxcd.io/blog/2022/10/september-2022-update/#flux-ecosystem

File: content/en/blog/2023-01-01-december-update/index.md [hunk 5]
```diff
@@ -0,0 +1,330 @@
...
+Our Flux Bug Scrubs still are happening on a weekly basis and remain one
+of the best ways to get involved in Flux. They are a friendly and
+welcoming way to learn more about contributing and how Flux is organised
+as a project.
+
+The next dates are going to be:
+
+- [2022-xx-yy 12:00 UTC, 14:00 CEST](https://www.meetup.com/weave-user-group/events/wvhvvsydclbnb/)
+- [2022-xx-yy 12:00 UTC, 14:00 CEST](https://www.meetup.com/weave-user-group/events/wvhvvsydclbgc/)
+
```
[the remark this answers is not in the document]
Comment (kingdonb) [reply content/en/blog/2023-01-01-december-update/index.md:191]: The bug scrubs are still ongoing, and we've spent some time lately looking at Weave GitOps and WG Enterprise features. I don't think we want to necessarily advertise here that people can come to us with WG inquiries, but that's a possibility for the future.

So I think I don't have anything specific to add to the write-up this time!

File: content/en/blog/2023-01-01-december-update/index.md [hunk 6]
```diff
@@ -0,0 +1,330 @@
...
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
```
[the remark this answers is not in the document]
Comment (chanwit) [reply content/en/blog/2023-01-01-december-update/index.md:87]: snowed under. Will do it shortly @dholbach

**Comment by dholbach** [comment]
Planned release: 2023-02-01.

File: content/en/blog/2023-01-01-december-update/index.md [hunk 7]
```diff
@@ -0,0 +1,319 @@
...
+### Soulé Ba joins Flux Core Maintainers
+
+Soulé Ba has been working on Flux for a long while. Already a maintainer
+of Flux's `go-git-providers`, he didn't stop there but was involved in
+a lot of the RFC planning process of many features and contributed code and
+fixes for a long long time.
+
+The Flux community is grateful to have you. Well deserved becoming a
+[https://github.com/fluxcd/community/pull/271](Core maintainer now), Soulé!
```
Comment (dholbach) [inline content/en/blog/2023-01-01-december-update/index.md:199]: @souleb Are you happy with this?

File: content/en/blog/2023-01-01-december-update/index.md [hunk 8]
```diff
@@ -0,0 +1,319 @@
...
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
+#### Terraform-controller
+
```
Comment (chanwit) [inline content/en/blog/2023-01-01-december-update/index.md]:
```suggestion

The team is currently working on getting the new release v0.14 out. They are updating the Terraform binary to version 1.3.7 and the Flux tool to version 0.38. Additionally, they are fixing the Helm chart and enabling the parallelism option for the apply stage. They are currently at release candidate v0.14.0-rc.2 with the new Helm chart version 0.10.0. Please stay tuned for further updates.
```

File: content/en/blog/2023-01-01-december-update/index.md [hunk 9]
```diff
@@ -0,0 +1,319 @@
...
+- "Since the new release of X, it supports Y." or
+- "Team lead X says: 'we have put a lot of effort into Y and are
+   really proud of the performance results' ..." or
+- "The team has been working on ..."
+
+-->
+
+#### Flux Subsystem for Argo
+
```
Comment (chanwit) [inline content/en/blog/2023-01-01-december-update/index.md]:
```suggestion

Flamingo is a tool that combines Flux and Argo CD to provide the best of both worlds for implementing GitOps on Kubernetes clusters. With Flamingo, you can:

- Automate the deployment of your applications to Kubernetes clusters and benefit from the improved collaboration and deployment speed and reliability that GitOps offers.
- Enjoy a seamless and integrated experience for managing deployments, with the automation capabilities of Flux embedded inside the user-friendly interface of Argo CD.
- Take advantage of additional features and capabilities that are not available in either Flux or Argo CD individually, such as the robust Helm support from Flux, Flux OCI Repository, Weave GitOps Terraform Controller for Infrastructure as Code, Weave Policy Engine, or Argo CD ApplicationSet for Flux-managed resources.

In recent releases, the team updated Flamingo to support Flux v0.38 and Argo CD v2.5.7, v2.4.19 and v2.3.13. Please note that Argo CD v2.2 will not be supported and updated by Flamingo anymore.

|Flux  | Argo CD | Image
|:----:|:-------:|---------------------------
|v0.38 | v2.5    | v2.5.7-fl.3-main-b0b6148f
|v0.38 | v2.4    | v2.4.19-fl.3-main-b0b6148f
|v0.38 | v2.3    | v2.3.13-fl.3-main-b0b6148f
|v0.37 | v2.2    | v2.2.16-fl.3-main-2bba0ae6
```

**Review by chanwit — CHANGES_REQUESTED** [review]
@dholbach could you please please commit these changes for me?

File: content/en/blog/2023-02-01-january-update/index.md [hunk 10]
```diff
@@ -0,0 +1,348 @@
...
+
+- CFP until 2023-02-05, [SustainabilityCon](https://events.linuxfoundation.org/cdcon-gitopscon/program/cfp/)
+  > May 10 – 12, 2023 | Vancouver, Canada
+  > Join the community of developers, technologists, sustainability leaders
+  > and anyone working on technological solutions to decarbonize the global
+  > economy, mitigate and address the impacts of climate change, and build a
+  > more sustainable future. SustainabilityCon provides a forum to drive open
+  > source innovation in energy efficiency and interoperability and clean
+  > development practices within industries ranging from manufacturing to
+  > agriculture and beyond through collaboration and learning within the community.
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:207]: @nikimanoledaki I thought I'd mention the CFP here as well - WDYT?

File: content/en/blog/2023-01-01-december-update/index.md (hunk 8 above)
Comment (chanwit) [reply content/en/blog/2023-01-01-december-update/index.md]: ^- @dholbach

File: content/en/blog/2023-01-01-december-update/index.md (hunk 9 above)
Comment (chanwit) [reply content/en/blog/2023-01-01-december-update/index.md]: ^- @dholbach

File: content/en/blog/2023-01-01-december-update/index.md (hunk 8 above)
Comment (dholbach) [reply content/en/blog/2023-01-01-december-update/index.md]: :pray:

File: content/en/blog/2023-01-01-december-update/index.md (hunk 9 above)
Comment (dholbach) [reply content/en/blog/2023-01-01-december-update/index.md]: :pray:

File: content/en/blog/2023-02-01-january-update/index.md [hunk 11]
```diff
@@ -0,0 +1,384 @@
...
+candidate v0.14.0-rc.2 with the new Helm chart version 0.10.0. Please stay
+tuned for further updates.
+
+#### Weave GitOps
+
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.15.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.14.1>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.13.0>
+- <https://github.com/weaveworks/weave-gitops/releases/tag/v0.12.0>
+
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:130]: @makkes @josefaworks Do you know who could help writing a bit of a Dec+Jan summary about what happened in Weave GitOps Core and Enterprise?

Here are past examples:

- https://fluxcd.io/blog/2022/12/november-2022-update/#flux-ecosystem
- https://fluxcd.io/blog/2022/11/october-2022-update/#flux-ecosystem
- https://fluxcd.io/blog/2022/10/september-2022-update/#flux-ecosystem

File: content/en/blog/2023-02-01-january-update/index.md [hunk 12]
```diff
@@ -0,0 +1,384 @@
...
+#### Azure GitOps
+
+#### VS Code GitOps Extension
+
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:134]: @juozasg Anything we should bring up here for Dec+Jan?

File: content/en/blog/2023-02-01-january-update/index.md [hunk 13]
```diff
@@ -0,0 +1,384 @@
...
+> support through this demo. Join us!
+
+Here is a list of additional videos and topics we really enjoyed -
+please let us know if we missed anything of interest and we will make
+sure to mention it in the next post!
+
+### Upcoming Events 📆
+
+We are happy to announce that we have a number of events coming up in
+February - tune in to learn more about Flux and GitOps best practices,
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:197]: @staceypotter @vanessaabankwah Anything we should bring up here for Dec+Jan+Feb?

File: content/en/blog/2023-02-01-january-update/index.md [hunk 14]
```diff
@@ -0,0 +1,384 @@
...
+### Soulé Ba joins Flux Core Maintainers
+
+Soulé Ba has been working on Flux for a long while. Already a maintainer
+of Flux's `go-git-providers`, he didn't stop there but was involved in
+a lot of the RFC planning process of many features and contributed code and
+fixes for a long long time.
+
+The Flux community is grateful to have you. Well deserved becoming a
+[https://github.com/fluxcd/community/pull/271](Core maintainer now), Soulé!
+
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:265]: @souleb Are you fine with us posting this? :-)

File: content/en/blog/2023-02-01-january-update/index.md (hunk 11 above)
Comment (dholbach) [reply content/en/blog/2023-02-01-january-update/index.md]: :pray:

File: content/en/blog/2023-02-01-january-update/index.md (hunk 10 above)
Comment (nikimanoledaki) [reply content/en/blog/2023-02-01-january-update/index.md:262]: LGTM! Thank you for the shoutout!

File: content/en/blog/2023-02-01-january-update/index.md (hunk 12 above)
Comment (juozasg) [reply content/en/blog/2023-02-01-january-update/index.md:153]: Unfortunately nothing new has been released yet. Slow progress. There is WGE integration work in very early stages.

**Review by makkes — APPROVED** [review]

File: content/en/blog/2023-02-01-january-update/index.md (hunk 13 above)
Comment (dholbach) [reply content/en/blog/2023-02-01-january-update/index.md:224]: https://www.meetup.com/gitops-community/events/289768516 maybe?

File: content/en/blog/2023-02-01-january-update/index.md [hunk 15]
```diff
@@ -0,0 +1,415 @@
...
+  multi-arch container images.
+- Add cert-manager Helm release to showcase how to install CRDs and custom
+  resources using `dependsOn`.
+- Add Let’s Encrypt ClusterIssuer to showcase how to patch resources in
+  production with Flux `Kustomization`.
+- Add the `flux-system` overlay to showcase how to configure Flux at
+  bootstrap time.
+
+♥ Big thanks to all the Flux contributors that helped us with this release!
+
```
Comment (dholbach) [inline content/en/blog/2023-02-01-january-update/index.md:60]: I would like to release the blog post tomorrow. Shall we give an outlook of the upcoming Flux release? c.f. https://github.com/fluxcd/flux2/issues/3533

cc @stefanprodan

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @dholbach 🥇

**Pull request merged**